REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
We can use Compat.jl to enable support for 0.4 going forward, or some other guards in the code.